public/pages/main/Main.tsx

```diff
@@ -70,7 +70,7 @@ export function Main(props: MainProps) {
       <EuiPageSideBar style={{ minWidth: 150 }} hidden={hideSideNavBar}>
         <EuiSideNav style={{ width: 150 }} items={sideNav} />
       </EuiPageSideBar>
-      <EuiPageBody>
+      <EuiPageBody restrictWidth={'70%'}>
```

Comment:
How about we remove this restriction to get wider page, especially for detector list

Reply:
Good point. It turns out that the min width of bottom 2 components `sunburst chart` + `detector feature list` is around 800px. And min width of Side Nav Bar is 150px. The smallest width which has no hidden contents is around 1000px.

Screenshot shows the above smallest width on mac 13in screen. 

<img width="647" alt="Screen Shot 2020-04-16 at 9 16 39 PM" src="https://user-images.githubusercontent.com/59710443/79531700-fb837e00-8027-11ea-9448-fb807800b66a.png">

Reply:
Is that smallest width acceptable?

Reply:
I think that's ok considering current popular resolution is above 1000px